package.json

```diff
@@ -8,7 +8,8 @@
     "watch": "gulp watch",
     "lint": "eslint .",
     "pretest": "npm-shrinkwrap-check -3 && eslint . && npm run compile && NODE_ENV=test npm run db-migrate",
-    "test": "NODE_ENV=test mocha && NODE_ENV=test npm run features",
+    "test-integration": "mocha",
```

Comment:
just want to be able to run all integration tests using npm, i.e:
`npm run integration-test`